hw/ip/csrng/doc/_index.md

```diff
@@ -6,12 +6,17 @@ title: "CSRNG HWIP Technical Specification"
 
 This document specifies the Cryptographically Secure Random Number Generator (CSRNG) hardware IP functionality.
 Due to the importance of secure random number generation (RNG), it is a topic which is extensively covered in security standards.
-This IP targets compliance with both [BSI's AIS31 recommendations for Common Criteria](https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Zertifizierung/Interpretationen/AIS_31_Functionality_classes_for_random_number_generators_e.pdf), as well as [NIST's SP 800-90](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-90Ar1.pdf) (as referenced in [FIPS 140-3](https://csrc.nist.gov/publications/detail/fips/140/3/final)).
+This IP targets compliance with both [BSI's AIS31 recommendations for Common Criteria](https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Zertifizierung/Interpretationen/AIS_31_Functionality_classes_for_random_number_generators_e.pdf), as well as [NIST's SP 800-90A](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-90Ar1.pdf) and [NIST's SP 800-90C (Second Draft)](https://csrc.nist.gov/CSRC/media/Publications/sp/800-90c/draft/documents/sp800_90c_second_draft.pdf), both of which are referenced in [FIPS 140-3](https://csrc.nist.gov/publications/detail/fips/140/3/final).
 Since these two standards use significantly different terminology, it is recommended that the reader refer to our RNG compliance strategy document for an overview of the various RNG classes and equivalencies between the two standards.
 The CSRNG IP supports both of these standards for both deterministic (DRNG) and true random number generation (TRNG).
-In NIST terms, it works with the [Entropy Source IP]({{< relref "hw/ip/entropy_src/doc" >}}) to satisfy the requirements as a DRBG or NRBG (non-deterministic random bit generator).
+In NIST terms, it works with the [Entropy Source IP]({{< relref "hw/ip/entropy_src/doc" >}}) to satisfy the requirements as a DRBG (deterministic random-bit-generator) or NRBG (non-deterministic random bit generator).
 In AIS31 language, this same implementation can be used to satisfy either the DRG.3 requirements for deterministic generation, or the PTG.3 requirements for cryptographically processed physical generation.
-This IP block is attached to the chip interconnect bus as a peripheral module conforming to the [Comportable guideline for peripheral functionality]({{< relref "doc/rm/comportability_specification" >}}), but also has direct hardware links to other IPs for secure internal transmission of random numbers.
+
+In this document the terms "DRNG" are "TRNG" are used most generally to refer to deterministic or true random number generation functionalities implemented to this specification.
```

Comment:
@imphil: this paragraph is new with this commit.  It would great if you could pound this into shape with your rigorous proofiing skils.